src/config.js

```diff
@@ -27,8 +36,37 @@ const Config = CoreObject.extend({
   },
 
   /**
-   * The apiKey for authenticating against shopify. This is your api client's
-   * public api token. Not the shared secret. Set during initialation.
+   * An object with keys for deprecated properties and values as functions that
+   * will transform the value into a usable value.
```

Comment:
I would mention the expected signature of any given transform is `value, configHash`.